.eslintrc.cli.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://json.schemastore.org/eslintrc",
+  "extends": [".eslintrc.json"],
+  "overrides": [
+    {
+      // This override adds type-checked rules.
+      "files": ["**/*.ts", "**/*.tsx"],
+      // Linting with type-checked rules is very slow and needs a lot of memory,
+      // so we exclude non-essential files.
+      "excludedFiles": ["examples/**/*", "test/**/*", "**/*.d.ts"],
+      "parserOptions": {
+        "project": true
```

Comment:
Does it always pick the nearest `tsconfig` automatically?

Reply:
[yes](https://typescript-eslint.io/blog/parser-options-project-true/)